modules/ROOT/pages/waf-rulesets.adoc

```diff
@@ -14,7 +14,7 @@ Request rulesets are divided into ten threat categories, which are called rulese
 * *Disable ruleset* - (Default) Ruleset detection is turned off.
 * *Detect and allow violations* - The violation is detected and you will get information in your logs at the INFO level, per incident. +
 They will also be included at the ERROR (default log level for Edge policies) level in a summary report. +
-See xref:1.2@runtime-fabric::enable-inbound-traffic#logs[the Runtime Fabric documentation].
+See xref:1.13@runtime-fabric::enable-inbound-traffic.adoc#logs[the Runtime Fabric documentation].
```

Comment:
```suggestion
See xref:runtime-fabric::enable-inbound-traffic.adoc#logs[the Runtime Fabric documentation].
```

I suspect this should always go to the latest version of the doc, but someone else would need to confirm.

Reply:
updated link as requested. Thank you!